containers/golang/Dockerfile

```diff
@@ -4,7 +4,8 @@ RUN apk update && apk add bash build-base
 
 ARG tool_type=rubenv-sql-migrate
 RUN if [ "$tool_type" = "rubenv-sql-migrate" ]; then \
-    go install github.com/rubenv/sql-migrate/...@latest; \
+    # CGO_CFLAGS is a temporary fix for https://github.com/mattn/go-sqlite3/pull/1177. We can remove this once rubenv-sql-migrate is updated to use the patched version of go-sqllite3.
+    CGO_CFLAGS="-D_LARGEFILE64_SOURCE" go install github.com/rubenv/sql-migrate/...@latest; \
   fi
```

Comment:
Just the link is sufficient. Also I believe you need a slash at the end to keep the expression going after a newline

Reply:
Wait, it's just a single line expression though and the comment wouldn't need a slash at the end.

Reply:
The whole if statement is a single line expression (notice the slashes at the end). Not sure what happens when it's concatenated even with the trailing slash, will everything after it be commented out then... Maybe just remove the comment altogether to be safe, not worth messing about redoing it just for that 🤷